packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -0,0 +1,104 @@
+import { useMutation } from '@tanstack/react-query';
+import { useCallback } from 'react';
+import { addDays, addHours } from 'date-fns';
+import {
+  setBookmarkReminder,
+  SetBookmarkReminderProps,
+} from '../../graphql/bookmarks';
+import { useToastNotification } from '../useToastNotification';
+
+export enum BookmarkReminderPreference {
+  OneHour = 'In 1 hour',
+  LaterToday = 'Later today',
+  Tomorrow = 'Tomorrow',
+  TwoDays = 'In 2 days',
+  NextWeek = 'Next week',
+}
```

Comment:
I've seen a lot of enum's usage in app, but what if we use and object here?

```ts

const BOOKMARK_REMINDER_PREFERENCE = {
  ONE_HOUR: 'In 1 hour',
  LATER_TODAY: 'Later today',
  TOMORROW: 'Tomorrow',
  TWO_DAYS: 'In 2 days',
  NEXT_WEEK: 'Next week',
} as const;

type BookmarkReminderPreference = typeof bookmarkReminderPreferences;
type BookmarkReminderPreferenceKeys = keyof BookmarkReminderPreference;

```

this would make easier iterate over options in modal/dropdown and centralize options for further implementations 👀

Reply:
enums are just objects in compiled javascript so you can do that as well

Reply:
Okay nevermind, never tried to Object.keys an enum 😱